model/client4.go

```diff
@@ -1497,6 +1497,72 @@ func (c *Client4) AssignBot(botUserId, newOwnerId string) (*Bot, *Response) {
 	return BotFromJson(r.Body), BuildResponse(r)
 }
 
```

Comment:
Surely `SetBotIconImage`, `DeleteBotIconImage` and `GetBotIconImage` should be exposed to plugins. Do you want to deal with this in this PR or in a different one? Or open a HW for this?

Reply:
Correct, planning on opening a HW ticket for that work, as noted in the PR description.

Reply:
Sorry, missed this in PR description. My bad.

Reply:
❤️